client/src/components/ElementEditor/ElementActions.js

```diff
@@ -74,7 +74,34 @@ class ElementActions extends Component {
    * @returns {ActionMenuComponent|null}
    */
   render() {
-    const { children, id, ActionMenuComponent } = this.props;
+    const { children, id, ActionMenuComponent, showEditTabs, showSaveAction } = this.props;
+
+    // Flatten the recursive structure of children - [[[..., react.element], react.element]
+    // Array.flat() isn't available for jest tests (or IE11)
+    const flatten = (arr) => {
+        let ret = [];
+        for (let i = 0; i < arr.length; i++) {
+            if (Array.isArray(arr[i])) {
+                ret = ret.concat(flatten(arr[i]));
+            } else {
+                ret.push(arr[i]);
+            }
+        }
+        return ret;
+    };
+
+    let kids = children;
+    if (children && Array.isArray(children)) {
+      kids = flatten(children).filter(child => {
+        if (!child || !child.props) {
+          return false;
+        }
+        if (child.props.title === 'Save') {
```

Comment:
Having a hardcoded string here isn't great, though I'm not sure a better way to do this

Maybe add a "showIfNonInlineEditable" property on AbstractAction?  That seems worse though IMO